api.planx.uk/modules/send/s3/index.test.ts

```diff
@@ -77,15 +77,18 @@ describe(`uploading an application to S3`, () => {
       .send({ payload: null })
       .expect(400)
       .then((res) => {
-        expect(res.body.error).toMatch(/Missing application payload/);
+        expect(res.body).toHaveProperty("issues");
+        expect(res.body).toHaveProperty("name", "ZodError");
```

Comment:
It's a known issue that these validation errors are not very human readable currently, but they're never really user facing.

Some ideas here on how we could improve this - https://zod.dev/?id=error-handling